### PR TITLE
start writing endpoints and refactor

### PR DESCRIPTION
**Pull request by tanneberger**

### CODE REVIEW

File: app/database.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 engine = create_engine(conn_string, echo=True)
 
-Model = declarative_base()
+base = declarative_base()
```
Comment (felbinger) [inline app/database.py]: `declarative_base()` returns a class, so it should be `Base`.

Comment (felbinger) [reply app/database.py]: https://docs.sqlalchemy.org/en/13/orm/extensions/declarative/api.html

File: app/models/user.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+from database import base, session
+
+
+class User(base):
+    __tablename__ = "user"
+
+    uuid = Column(UUIDColumn(as_uuid=True), primary_key=True, index=True, default=uuid4)
+    created_at = Column(DateTime, default=datetime.utcnow)
+
+    devices = relationship("Device", back_populates="user", passive_deletes=True)
```
Comment (felbinger) [inline app/models/user.py:19]: How does this identify the user? I would expect something like user_uuid?

**Review by felixevers — DISMISSED** [review]

File: app/utils/jwt.py [hunk 3]
```diff
@@ -8,15 +9,20 @@
...
+from models.user import User
 
 
 def generare_secret_key() -> str:
-    binascii.b2a_hex(os.urandom(32))
+    return binascii.b2a_hex(os.urandom(32))
 
 
 def get_secret_key() -> str:
     # this sets a default value for the SECRET_KEY_FILE
```
Comment (felbinger) [inline app/utils/jwt.py:23]: it does not set a default value? Comment can be misinterpreted...

File: app/models/user.py (hunk 2 above)
Comment (felixevers) [reply app/models/user.py:19]: Thats stupid!

File: docker-compose.yml [hunk 4]
```diff
@@ -4,7 +4,10 @@ services:
   postgres:
     image: postgres
     restart: always
-    env_file: .postgres.env
+    environment:
```
Comment (felbinger) [inline docker-compose.yml:7]: In my mind the env_file is way better, but accepted.

Comment (felbinger) [reply docker-compose.yml:7]: Please delete .api.env completely

Comment (felixevers) [reply docker-compose.yml:7]: NO!

**Pull request merged**